ios/FluentUI/Notification/NotificationView_SwiftUI.swift

```diff
@@ -45,6 +45,7 @@ public struct NotificationViewSwiftUI: View {
     @Environment(\.swiftUIInsets) private var safeAreaInsets: EdgeInsets
     @Environment(\.theme) var theme: FluentUIStyle
     @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
+    @Environment(\.viewController) var viewController: UIViewController?
```

Comment:
I see how you're reading from this variable below, but I don't see where it's set. Does that mean the code is currently always using the default value?